File: linguagens/Python/Exercicios/terreno.py
largura: int 
comprimento: int
valor: int
area: int
preco: int   

largura = int(input("Digite a largura do seu terreno: "))
comprimento = int(input("DIgite o comprimento do se terreno: "))
valor = int(input("Digite o valor do metro quadrado: "))

area = largura * comprimento

preco = valor * area

print(f"Area do terreno: {area:.2f}")

print(f"Preço do terreno: {preco:.2f}")

File: linguagens/C/exercicios/terreno.c
#include <stdio.h>
#include <string.h>


int main()
{
    
    double largura, comprimento, valor, area, preco; 
    
    printf("Digite a largura: ");
    scanf ("%lf", &largura); 
    
    printf("Digite a comprimento: ");
    scanf ("%lf", &comprimento);
    
    printf("Digite a valor em metro quadrado: ");
    scanf ("%lf", &valor);
    
    
    area = largura * comprimento; 
    
    printf("Area é = %.2lf\n",  area); 
    
    preco = area * valor; 
    
    printf("preco é = %.2lf" , preco);
    
	return 0;
}